tests/by-util/test_df.rs

```diff
@@ -279,15 +298,7 @@ fn test_output_file_specific_files() {
         .succeeds()
         .stdout_move_str();
     let actual: Vec<&str> = output.lines().collect();
-    assert_eq!(
-        actual,
-        vec![
-            "File            ",
-            "a               ",
-            "b               ",
-            "c               "
-        ]
-    );
+    assert_eq!(actual, vec!["File", "a   ", "b   ", "c   "]);
```

Comment:
This is certainly an improvement but it doesn't quite match the behavior of GNU. The last column in the table does not have trailing spaces.

This is certainly an improvement so I'm okay with merging this as-is, but just bringing it up as another opportunity to improve the output of `df`.

Reply:
Hm, I think you misread the snippet above. It contains a single column with a header and three values. And the trailing spaces are there because those values are shorter than the header.

Reply:
Yes, I understand. When I said "the last column" I should have said "the last and only column in this test case". What I meant is that in GNU df there are no trailing spaces in the rightmost column of the output table:
```
$ touch a && df --output=file a | tr " " "_"
File
a
```

But in this branch there are trailing spaces:
```
$ touch a && ./target/debug/df --output=file a | tr " " "_"
File
a___
```

I'm okay with it, I just wanted to mention it as a follow-up improvement for a future pull request.

Reply:
Thanks for the explanation, that makes sense.